pkg/distro/defs/README.md

```diff
@@ -113,6 +119,7 @@ shared and merge via the  `<<:` operation.
 The `when` part of the condition can contain one or more
 of:
 - distro_name
+- `not_distro_name`
```

Comment:
Why is this in backticks unlike the others?

Reply:
I put all into backticks now, my editor was making the _distro_ part of not_distro_name italic, but maybe that is just an artifact and GH is rendering it fine I did not double check.